index.html

```diff
@@ -11,13 +11,38 @@
         height: 100%;
       }
 
+      #controls {
+        width: inherit;
+        height: 20px;
+        padding:10px;
+
+        display: flex;
+        flex-direction: row;
+      }
+
+      #controls h4 {
+        margin:2px;
+        margin-right: 20px;
+      }
+
       #voyager-embed {
         width: inherit;
         height: inherit;
       }
     </style>
   </head>
   <body>
+    <div id="controls">
```

Comment:
I think it might be better to add this to the menu bar.  Additional header will look pretty weird after we have proper Voyager header (soon this week).